VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md

```diff
@@ -10,7 +10,7 @@ In order to start creating or modifying tables with VPE, the first thing you'll
 > [!NOTE]
 > As long as you don't use Unity for a game that makes $100K or more in revenue or funding a year, the free [Personal](https://store.unity.com/compare-plans) plan is sufficient for you.
 
-Unity uses an application called *Unity Hub* to update itself and provide quick access to your projects. The install process is straight-forward and documented [here](https://docs.unity3d.com/Manual/GettingStartedInstallingHub.html) if you run into troubles.
+Unity uses an application called *Unity Hub* to update itself, create new projects and provide quick access to them. Choose version 3 Beta or newer as we need it for later. The install process is straight-forward and documented [here](https://docs.unity3d.com/Manual/GettingStartedInstallingHub.html) if you run into troubles.

```

Comment:
You sure that Hub v3 is necessary?

Reply:
No, not sure. I just assumed to use the latest as I saw it on one of the screenshots.